src/controllers/ReviewPenaltyController.ts

```diff
@@ -0,0 +1,104 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { controller } from 'inversify-express-utils';
+import { SafeNavigationBaseController } from './SafeNavigationBaseController';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { Appeal } from 'app/models/Appeal';
+import { OTHER_REASON_DISCLAIMER_PAGE_URI, PENALTY_DETAILS_PAGE_URI, REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'review-penalty';
+const navigation = {
+    previous(): string {
+        return PENALTY_DETAILS_PAGE_URI;
+    },
+    next(): string {
+        return OTHER_REASON_DISCLAIMER_PAGE_URI;
+    }
+};
+
+type TableColumn = {
+    text: string;
+};
+type TableRow = TableColumn[];
+type Table = {
+    caption: string,
+    header: TableRow;
+    madeUpToDate: string;
+    tableRows: TableRow[];
+};
```

Comment:
Could we perhaps move this to somewhere in the models directory, maybe its own file? I think this adds noise to the controller.